packages/engine-server/src/backup/service.ts

```diff
@@ -68,8 +69,8 @@ export class BackupService implements Disposable, IBackupService {
       root: this.wsRoot,
     });
     await fs.ensureDir(this.backupRoot);
-    Object.keys(BackupKeyEnum).forEach(async (key) => {
-      await fs.ensureDir(path.join(this.backupRoot, key));
+    await asyncLoop<string>(Object.values(BackupKeyEnum), (key) => {
```

Comment:
Could we add a docstring to `BackupKeyEnum` to say whether it's the key or the value that's supposed to be the path? The switch from the key to value confused me when reviewing at least